databricks/koalas/tests/test_stats.py

```diff
@@ -99,6 +99,19 @@ def test_cov_corr_meta(self):
             # assert ddf.a.cov(ddf.b)._meta.dtype == 'f8'
             # assert ddf.a.corr(ddf.b)._meta.dtype == 'f8'
 
+    def test_stats_on_boolean(self):
+        df = pd.DataFrame({'A': [True, False]})
```

Comment:
If you test the case of this `df`, asserts of `var()` and `std()` will fail. 
```python
        df = pd.DataFrame({'A': [True, False,True],
                           'B': [False,False,True]})
```

Reply:
Thanks for your feedback. The error seems to be in Koalas' `assert_eq()` method. When using pandas' `pd.testing.assert_series_equal()` instead, the tests for `var()` and `std()` work as expected. However, this revealed another issue where `min()` and `max()` return boolean values for pandas and integers for Koalas which I have now fixed in 5992c6f.